sense-android-library/documentation/sensor_profiles.md

```diff
@@ -0,0 +1,472 @@
+    },
+    "z-axis": {
+      "description": "The acceleration force applied on the z-axis in m/s2",
+      "type": "number"
+    }
+  },
+  "required": [
+    "x-axis",
+    "y-axis",
+    "z-axis"
+  ]
+}
+~~~
+
+### battery
+*renamed from battery_sensor*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "status": {
+      "description": "The status of the battery, e.g. charging, discharging, full",
+      "type": "string"
+    },
+    "level": {
+      "description": "The battery level in percentage.",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "level"
+  ]
+}
+~~~
+### call
+*renamed from call_state*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "state": {
+      "description": "The state of the phone",
+      "enum": ["idle", "dialing", "ringing", "calling"]
+    },
+    "incomingNumber": {
+      "description": "The phone number of the in comming call",
+      "type": "string"
+    },
+    "outgoingNumber": {
+      "description": "The phone number of the out going call",
+      "type": "string"
+    }
+  },
+  "required": [
+    "state"
+  ]
+}
+~~~
+### light
+
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The illuminance in lx",
+  "type": "number"
+}
+~~~
+
+*removed inner object with key lux*
+
+### noise
+*renamed from noise_sensor*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The Ambient noise in decibel",
+  "type": "number",
+}
+~~~
+
+### position
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "longitude": {
+      "description": "The longitude in degrees",
+      "type": "number"
+    },
+    "latitude": {
+      "description": "The latitude in degrees",
+      "type": "number"
+    },
+    "altitude": {
+      "description": "altitude in meters above the WGS 84 reference ellipsoid.",
+      "type": "number"
+    },
+    "accuracy": {
+      "description": "accuracy in meters",
+      "type": "number"
+    },
+    "speed": {
+      "description": "The speed in meters/second over ground.",
+      "type": "number"
+    },
+    "bearing": {
+      "description": "The average bearing in degrees",
+      "type": "number"
+    },
+    "provider": {
+      "description": "The location provider, e.g. GPS, NETWORK or FUSED",
+      "type": "string"
+    }
+  },
+  "required": [
+    "longitude",
+    "latitude",
+    "accuracy",
+    "provider"
+  ]
+}
+~~~
+
+### proximity
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The proximity to an object in cm",
+  "type": "number",
+}
+~~~
+
+### screen
+*renamed from screen_activity*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The status of the screen, e.g. on or off",
+  "enum": [ "on", "off"]
```

Comment:
@orhan89 I think a sensor with name "screen" and values  true and false, is not that descriptive, could also mean that there is a screen or not, and with the enum we could define other states like: idle, active. But we could use a boolean and rename the sensor if that is more clear. What do you think?